### PR TITLE
[ToricVarieties] Add total_space(E)

### PR DESCRIPTION
**Pull request by mgemath**
Here we add the function total_space(E::ToricLineBundle...).
It return, as a toric variety, the total space of the sum of the line bundles appearing in E.

### CODE REVIEW

File: test/AlgebraicGeometry/ToricVarieties/total_space.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	end
+
+	@testset "Test that omega_S is Calabi-Yau for any S smooth surface" begin
+		@testset "S is Hirzebruch" begin
+			for a in 0:10
+				S = hirzebruch_surface(NormalToricVariety, a; set_attributes)
+				X = total_space(canonical_bundle(S))
+				@test is_smooth(X) == true
+				@test !is_fano(X)
+        @test !is_complete(X)
```
Comment (fingolfin) [inline test/AlgebraicGeometry/ToricVarieties/total_space.jl]: Indention is weird here. I think you are mixing tabs and spaces. Please do not use tabs. In fact, please ideally read our [notes about code formatting in OSCAR](https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Code-formatting) 😄

Comment (mgemath) [reply test/AlgebraicGeometry/ToricVarieties/total_space.jl]: Thank you very much! I corrected the commit following your remark.

Comment (HereAround) [reply test/AlgebraicGeometry/ToricVarieties/total_space.jl]: It seemed that there was still a mixture of tabs and spaces. If the editorconfig is correctly acknowledged by VSCode, pressing tab should add two white space characters. (For me, they show as two centered dots, whereas a tab is shown as an arrow to the right).

I have taken the liberty to update your branch with a change, that turns all tabs into two whitespace characters. If I did not make a mistake, this should now agree with the requirements in https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Code-formatting.

Comment (HereAround) [reply test/AlgebraicGeometry/ToricVarieties/total_space.jl]: I am very sorry @mgemath , in my attempt to help I must have overwritten part (hopefully just a small part) of your desired changes. Really sorry. I just tried to fix my mistake, but could not. Maybe you immediately see what change I overwrote?

Comment (mgemath) [reply test/AlgebraicGeometry/ToricVarieties/total_space.jl]: Do not worry @HereAround. You were trying to correct my own mistakes. So it is me who must apologize in the first place. Anyway, it seems that you undone some of my corrections. This is the list.

file: src/AlgebraicGeometry/ToricVarieties/Proj/constructors.jl
lines: 28, 32, 39, 43, 60, 65.

file: src/exports.jl
line: 1378.

file: test/AlgebraicGeometry/ToricVarieties/proj.jl
line: 43.

You can find the changes here: https://github.com/oscar-system/Oscar.jl/compare/8eb7eb4cfc9d1aca1f80cbbe76f129b5987e6d06..b68e44e915573330cea170b104528d7a5cb8c595

Comment (HereAround) [reply test/AlgebraicGeometry/ToricVarieties/total_space.jl]: Thank you @mgemath. Just corrected my earlier mistake and added one more commit on top with a few minor optimizations. Let us see if the tests now pass.

**Review by HereAround — APPROVED** [review]
Thank you @gmemath. This looks good to me.

As discussed above, I adjusted the tabs vs whitespaces issue. We can discuss more on slack if this remains to be an issue. I have also done a few minor adjustments - nothing major.

Long story short: Thank you for the added functionality. Great!

**Pull request merged**